src/colorizer/utils/math_utils.ts

```diff
@@ -49,7 +51,12 @@ export function remap(
   clamp: boolean = true
 ): number {
   if (clamp) {
-    input = Math.min(Math.max(input, inMin), inMax);
+    const min = Math.min(inMin, inMax);
+    const max = Math.max(inMin, inMax);
+    input = Math.min(Math.max(input, min), max);
+  }
+  if (inMin === inMax) {
+    return outMin;
```

Comment:
Nit: a couple of tiny simple unit tests might be nice here, not sure if they already exist. (regular in-bounds case, out of bounds case, reversed range in and out of bounds cases, and input===inMin or inMax... clamp/no clamp for all).

Reply:
They do! ✨✨